package.json

```diff
@@ -74,7 +74,7 @@
     "@angular/platform-browser-dynamic": "^11.2.14",
     "@angular/router": "^11.2.14",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "6.13.10-case-file-view-display-document-upload-date-v4",
+    "@hmcts/ccd-case-ui-toolkit": "6.13.10-case-file-view-sort-by-document-upload-date",
```

Comment:
@johnbenjamin-hmcts - shouldn't this be `6.13.10-case-file-view-sort-by-document-upload-date-v2`?

Reply:
@Daniel-Lam - yes, I published the toolkit version then noticed that I have to get the latest from Release and publish a new version.